### ISSUE TITLE
Drop unused code from `src/_includes/code`

### ISSUE THREAD

**Issue by chalin** [flutter/website#1238]
@filiph @sethladd @mit-mit @Hixie - I'd like to **drop all unused code** snippets from `src/_includes/code`, which means all but 4 files from `src/_includes/code/layout`. Any objections?

TL;DR

How much of the code under [src/_includes/code](https://github.com/flutter/website/tree/dash/src/_includes/code) is _actually_ used in the website?

If I grep for `{% include includelines ... %}` I find only 4 files being used under `src/tutorials/layout` (the other two are the `includelines` template itself):

```console
> grep -R includelines src
src/_includes/includelines:  {% include includelines filename=PATH start=INT count=INT %}
src/_includes/includelines:  {% include includelines filename='src/HelloWorld.java' start=10 count=5 %}
src/tutorials/layout/index.md:{% include includelines filename="code/layout/row/main.dart" start=40 count=8 %}
src/tutorials/layout/index.md:{% include includelines filename="code/layout/column/main.dart" start=40 count=8 %}
src/tutorials/layout/index.md:{% include includelines filename="code/layout/row-expanded/main.dart" start=40 count=15 %}
src/tutorials/layout/index.md:{% include includelines filename="code/layout/row-expanded-2/main.dart" start=40 count=14 %}
```

I'd like to delete code that isn't included, any objections?

cc @Sfshaza @gspencergoog

**Comment by filiph** [comment]
SGTM

**Comment by Sfshaza** [comment]
Let me take a look, @chalin. I thought we were using a lot of that code.

**Comment by Hixie** [comment]
I must abstain, I don't know anything about that directory.

**Comment by mit-mit** [comment]
I have no knowledge about this either

**Comment by chalin** [comment]
Closed by #1245 and #1237
